Units/parser-ruby.r/ruby-block-assign.d/expected.tags

```diff
@@ -0,0 +1,6 @@
+Bar	input.rb	/^c = class Bar$/;"	c
+Foo	input.rb	/^m = module Foo$/;"	m
+method_a	input.rb	/^  x = def method_a$/;"	f	class:Bar
+method_b	input.rb	/^  def method_b$/;"	f	class:Bar
+method_c	input.rb	/^  def method_c$/;"	f	class:Bar
+method_d	input.rb	/^  def method_d$/;"	f	class:Bar
```

Comment:
Shouldn't `m`, `c` and `x` be reported as something (variables I guess)?

I'm not sure where this syntax is used, but apparently it does something (store the result of the last statement? not sure from experimenting…), so I guess they should ideally be reported, shouldn't they?

Reply:
They could be reported as variables, but I believe this hasn't been the case thus far. Anyway, it isn't in the scope of this PR, as it just aims to fix the annoying bug where class members are reported as free-floating methods if block assignment is used.